src/interactions/abstractInteraction.ts

```diff
@@ -17,6 +17,34 @@ export module Interaction {
       this._componentToListenTo = component;
       this._hitBox = hitBox;
     }
+
+    /**
+     * Translates an <svg>-coordinate-space point to Component-space coordinates.
+     *
+     * @param {Point} p A Point in <svg>-space coordinates.
+     *
+     * @return {Point} The same location in Component-space coordinates.
+     */
+    protected _translateToComponentSpace(p: Point): Point {
+      var origin = this._componentToListenTo.originToSVG();
```

Comment:
Aside - After reading this usage, I thought the return variable would be an SVG but nope it's the origin for the SVG.  While this is an API-breaking change, would svgOrigin be a bad name?